[2019-10-16 15:05:39,443] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [queued]>
[2019-10-16 15:05:39,452] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [queued]>
[2019-10-16 15:05:39,452] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-10-16 15:05:39,452] {taskinstance.py:839} INFO - Starting attempt 1 of 5
[2019-10-16 15:05:39,453] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-10-16 15:05:39,460] {taskinstance.py:859} INFO - Executing <Task(BashOperator): print_first_ten_lines> on 2019-10-16T09:30:00+00:00
[2019-10-16 15:05:39,461] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'file_operations_pipeline', 'print_first_ten_lines', '2019-10-16T09:30:00+00:00', '--job_id', '13', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/file_mover.py', '--cfg_path', '/tmp/tmp7kgn7vfe']
[2019-10-16 15:05:40,103] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines /home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/configuration.py:627: DeprecationWarning: You have two airflow.cfg files: /home/sat/airflow/airflow.cfg and /home/sat/Projects/p1/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /home/sat/Projects/p1/airflow.cfg, and you should remove the other file
[2019-10-16 15:05:40,103] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   category=DeprecationWarning,
[2019-10-16 15:05:40,187] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines [2019-10-16 15:05:40,186] {settings.py:213} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=15027
[2019-10-16 15:05:40,400] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines [2019-10-16 15:05:40,399] {__init__.py:51} INFO - Using executor CeleryExecutor
[2019-10-16 15:05:40,826] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines [2019-10-16 15:05:40,825] {dagbag.py:90} INFO - Filling up the DagBag from /home/sat/Projects/p1/dags/file_mover.py
[2019-10-16 15:05:40,851] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines [2019-10-16 15:05:40,851] {cli.py:516} INFO - Running <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [running]> on host sat-Inspiron-5559
[2019-10-16 15:05:40,863] {bash_operator.py:81} INFO - Tmp dir root location: 
 /tmp
[2019-10-16 15:05:40,864] {bash_operator.py:91} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=file_operations_pipeline
AIRFLOW_CTX_TASK_ID=print_first_ten_lines
AIRFLOW_CTX_EXECUTION_DATE=2019-10-16T09:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-16T09:30:00+00:00
[2019-10-16 15:05:40,864] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmpgf29088w/print_first_ten_lines4boca6ff
[2019-10-16 15:05:40,864] {bash_operator.py:115} INFO - Running command: head -n 10 target_dir/source_dir/AB_NYC_2019.csv
[2019-10-16 15:05:40,869] {bash_operator.py:124} INFO - Output:
[2019-10-16 15:05:40,871] {bash_operator.py:128} INFO - head: cannot open 'target_dir/source_dir/AB_NYC_2019.csv' for reading: No such file or directory
[2019-10-16 15:05:40,871] {bash_operator.py:132} INFO - Command exited with return code 1
[2019-10-16 15:05:40,879] {taskinstance.py:1051} ERROR - Bash command failed
Traceback (most recent call last):
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 136, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2019-10-16 15:05:40,880] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-10-16 15:05:40,890] {logging_mixin.py:95} INFO - [[34m2019-10-16 15:05:40,890[0m] {[34mconfiguration.py:[0m299} WARNING[0m - section/key [[1msmtp[0m/[1msmtp_user[0m] not found in config[0m
[2019-10-16 15:05:40,891] {taskinstance.py:1086} ERROR - Failed to send email to: ['satish.kumar@marketale.com']
[2019-10-16 15:05:40,891] {taskinstance.py:1087} ERROR - [Errno 111] Connection refused
Traceback (most recent call last):
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 136, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1076, in handle_failure
    self.email_alert(error)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1307, in email_alert
    send_email(self.task.email, subject, html_content)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/utils/email.py", line 55, in send_email
    mime_subtype=mime_subtype, mime_charset=mime_charset, **kwargs)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/utils/email.py", line 101, in send_email_smtp
    send_MIME_email(smtp_mail_from, recipients, msg, dryrun)
  File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/utils/email.py", line 121, in send_MIME_email
    s = smtplib.SMTP_SSL(SMTP_HOST, SMTP_PORT) if SMTP_SSL else smtplib.SMTP(SMTP_HOST, SMTP_PORT)
  File "/usr/lib/python3.6/smtplib.py", line 251, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/lib/python3.6/smtplib.py", line 336, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/lib/python3.6/smtplib.py", line 307, in _get_socket
    self.source_address)
  File "/usr/lib/python3.6/socket.py", line 724, in create_connection
    raise err
  File "/usr/lib/python3.6/socket.py", line 713, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
[2019-10-16 15:05:40,902] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines Traceback (most recent call last):
[2019-10-16 15:05:40,902] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/bin/airflow", line 32, in <module>
[2019-10-16 15:05:40,902] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     args.func(args)
[2019-10-16 15:05:40,902] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     return f(*args, **kwargs)
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/bin/cli.py", line 522, in run
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     _run(args, dag, ti)
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     pool=args.pool,
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     return func(*args, **kwargs)
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     result = task_copy.execute(context=context)
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines   File "/home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 136, in execute
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines     raise AirflowException("Bash command failed")
[2019-10-16 15:05:40,903] {base_task_runner.py:115} INFO - Job 13: Subtask print_first_ten_lines airflow.exceptions.AirflowException: Bash command failed
[2019-10-16 15:05:44,459] {logging_mixin.py:95} INFO - [[34m2019-10-16 15:05:44,456[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
[2019-10-16 15:08:04,677] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [queued]>
[2019-10-16 15:08:04,692] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [queued]>
[2019-10-16 15:08:04,692] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-10-16 15:08:04,692] {taskinstance.py:839} INFO - Starting attempt 1 of 5
[2019-10-16 15:08:04,692] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-10-16 15:08:04,704] {taskinstance.py:859} INFO - Executing <Task(BashOperator): print_first_ten_lines> on 2019-10-16T09:30:00+00:00
[2019-10-16 15:08:04,705] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'file_operations_pipeline', 'print_first_ten_lines', '2019-10-16T09:30:00+00:00', '--job_id', '6', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/file_mover.py', '--cfg_path', '/tmp/tmpg4lqohir']
[2019-10-16 15:08:05,347] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines /home/sat/Projects/p1/avenv/lib/python3.6/site-packages/airflow/configuration.py:627: DeprecationWarning: You have two airflow.cfg files: /home/sat/airflow/airflow.cfg and /home/sat/Projects/p1/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read /home/sat/Projects/p1/airflow.cfg, and you should remove the other file
[2019-10-16 15:08:05,348] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines   category=DeprecationWarning,
[2019-10-16 15:08:05,423] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines [2019-10-16 15:08:05,423] {settings.py:213} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=15505
[2019-10-16 15:08:05,624] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines [2019-10-16 15:08:05,623] {__init__.py:51} INFO - Using executor CeleryExecutor
[2019-10-16 15:08:06,071] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines [2019-10-16 15:08:06,070] {dagbag.py:90} INFO - Filling up the DagBag from /home/sat/Projects/p1/dags/file_mover.py
[2019-10-16 15:08:06,097] {base_task_runner.py:115} INFO - Job 6: Subtask print_first_ten_lines [2019-10-16 15:08:06,097] {cli.py:516} INFO - Running <TaskInstance: file_operations_pipeline.print_first_ten_lines 2019-10-16T09:30:00+00:00 [running]> on host sat-Inspiron-5559
[2019-10-16 15:08:06,109] {bash_operator.py:81} INFO - Tmp dir root location: 
 /tmp
[2019-10-16 15:08:06,110] {bash_operator.py:91} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_ID=file_operations_pipeline
AIRFLOW_CTX_TASK_ID=print_first_ten_lines
AIRFLOW_CTX_EXECUTION_DATE=2019-10-16T09:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-16T09:30:00+00:00
[2019-10-16 15:08:06,110] {bash_operator.py:105} INFO - Temporary script location: /tmp/airflowtmpe2bm2hme/print_first_ten_linesjc9dkb96
[2019-10-16 15:08:06,111] {bash_operator.py:115} INFO - Running command: head -n 10 /home/sat/Projects/p1/target_dir/source_dir/AB_NYC_2019.csv
[2019-10-16 15:08:06,117] {bash_operator.py:124} INFO - Output:
[2019-10-16 15:08:06,120] {bash_operator.py:128} INFO - id,name,host_id,host_name,neighbourhood_group,neighbourhood,latitude,longitude,room_type,price,minimum_nights,number_of_reviews,last_review,reviews_per_month,calculated_host_listings_count,availability_365
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 2539,Clean & quiet apt home by the park,2787,John,Brooklyn,Kensington,40.64749,-73.97237,Private room,149,1,9,2018-10-19,0.21,6,365
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 2595,Skylit Midtown Castle,2845,Jennifer,Manhattan,Midtown,40.75362,-73.98377,Entire home/apt,225,1,45,2019-05-21,0.38,2,355
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 3647,THE VILLAGE OF HARLEM....NEW YORK !,4632,Elisabeth,Manhattan,Harlem,40.80902,-73.9419,Private room,150,3,0,,,1,365
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 3831,Cozy Entire Floor of Brownstone,4869,LisaRoxanne,Brooklyn,Clinton Hill,40.68514,-73.95976,Entire home/apt,89,1,270,2019-07-05,4.64,1,194
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 5022,Entire Apt: Spacious Studio/Loft by central park,7192,Laura,Manhattan,East Harlem,40.79851,-73.94399,Entire home/apt,80,10,9,2018-11-19,0.10,1,0
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 5099,Large Cozy 1 BR Apartment In Midtown East,7322,Chris,Manhattan,Murray Hill,40.74767,-73.975,Entire home/apt,200,3,74,2019-06-22,0.59,1,129
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 5121,BlissArtsSpace!,7356,Garon,Brooklyn,Bedford-Stuyvesant,40.68688,-73.95596,Private room,60,45,49,2017-10-05,0.40,1,0
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 5178,Large Furnished Room Near B'way ,8967,Shunichi,Manhattan,Hell's Kitchen,40.76489,-73.98493,Private room,79,2,430,2019-06-24,3.47,1,220
[2019-10-16 15:08:06,121] {bash_operator.py:128} INFO - 5203,Cozy Clean Guest Room - Family Apt,7490,MaryEllen,Manhattan,Upper West Side,40.80178,-73.96723,Private room,79,2,118,2017-07-21,0.99,1,0
[2019-10-16 15:08:06,121] {bash_operator.py:132} INFO - Command exited with return code 0
[2019-10-16 15:08:09,672] {logging_mixin.py:95} INFO - [[34m2019-10-16 15:08:09,669[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 0[0m
